.github/scripts/snyk-report.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -e
+
+KEYCLOAK_REPO="keycloak/keycloak"
+
+# Prevent duplicates by checking if a similar CVE exists
+check_github_issue_exists() {
+    local issue_title="$1"
+    # Extract the string before the last hyphen
+    local extracted_title="${issue_title%-*}"
+    
+    # Encode the extracted title for the URL
+    local encoded_title=$(echo "$extracted_title" | jq -sRr @uri)
+    local search_url="https://api.github.com/search/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+$encoded_title+repo:$KEYCLOAK_REPO"
```

Comment:
Is it Ok if the issue exists but closed? What happens if we decide to no fix it? aybe we should add something for those cases. I don't want to have issues re-openen all teh time. :smile: